src/styles/BookList.module.css

```diff
@@ -5,34 +5,108 @@
 }
 
 .book {
-  border: 1px solid black;
+  border: 1px solid rgb(211, 207, 207);
   margin: 1rem;
   list-style-type: none;
   padding: 1rem;
-  border-radius: 5px;
+  height: 27vh;
+  border-radius: 4px;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
 }
 
 .btnContainer {
   margin-top: 10px;
 }
 
 button {
-  margin-right: 0.25rem;
+  margin-right: 2rem;
+  color: #0290ff;
+  border: none;
+  background: transparent;
 }
 
 form {
-  margin: 2rem 1rem;
+  padding: 0 4rem 4rem 4rem;
+  background-color: rgb(200, 200, 200, 0.1);
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 1rem;
+}
+
+.bookBtn {
+  background-color: #0290ff;
+  color: white;
+  padding: 0.7rem;
+  border-radius: 3px;
+}
+
+.bookBtn:hover {
+  background-color: #7fa5c3;
 }
 
 label {
-  display: block;
+  color: grey;
+  font-weight: bold;
+  font-size: 1.3rem;
 }
 
 input {
+  width: 30vw;
+  height: 3vh;
   margin-right: 1rem;
-  padding: 2px;
+  padding: 1rem;
+  font-size: 1rem;
 }
 
-ul {
+.body {
   display: block;
+  background-color: rgb(200, 200, 200, 0.1);
+  padding: 2rem;
+}
+
+.circle {
+  border: 5px solid #0290ff;
+  width: 4rem;
+  height: 4rem;
+  margin-right: 1rem;
+  border-radius: 50%;
+}
+
+small {
+  color: rgb(155, 152, 152);
+}
+
+.blue {
```

Comment:
- [ ] Kindly ensure that the following highlighted elements in the image have the same color as in the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) as per the requirements

<img width="1430" alt="Screenshot 2023-06-15 at 22 33 49" src="https://github.com/forLoop94/book-store/assets/64774936/6ca8979e-0b4a-4a7b-b5c9-3006ce26c9db">